nx-X11/config/cf/Imakefile

```diff
@@ -32,115 +31,24 @@ nxversion.def
 
 RULEFILES = \
 Imake.rules \
-Motif.rules \
-Win32.rules \
 X11.rules \
-bsdLib.rules \
```

Comment:
Are you sure we want to do that? I figure this will make it impossible to build nxagent on *BSD... it's probably not a too big deal because no one did so far, but do we really want to disable it completely?